packages/client/src/pages/warehouse.vue

```diff
@@ -7,55 +7,16 @@
       >
         <template #side-actions>
           <q-btn
-            v-if="screenWidth === WidthSize.SM"
-            :icon="mdiDotsVertical"
-            color="black-54"
-            round
-            flat
-          >
-            <q-menu>
-              <q-item clickable @click="swapView()">
-                <q-item-section>
-                  <q-item-label>
-                    {{
-                      t(
-                        `warehouse.${isSortedByCopyCode ? "sortByISBN" : "sortByCopyCode"}`,
-                      )
-                    }}
-                  </q-item-label>
-                </q-item-section>
-              </q-item>
-
-              <q-item clickable @click="checkOtherWarehouse()">
-                <q-item-section>
-                  <q-item-label>
-                    {{ t("warehouse.checkOtherWarehouse", [otherCityName]) }}
-                  </q-item-label>
-                </q-item-section>
-              </q-item>
-            </q-menu>
```

Comment:
Ok, but let's leave here a TODO: stating that we need to add the possibility to view stuff from the other retail location.